Patterns/Chinese/Chinese-URL.yaml

```diff
@@ -0,0 +1,330 @@
+---
+ProtocolRegex: !simpleRegex
+  def: ((https?|ftp):\/\/)
+PortRegex: !simpleRegex
+  def: (:\d{1,5})
+ExtractionRestrictionRegex: !simpleRegex
+  def: (?<=\s|[\'""\(\[:：]|^|[\u0800-\u9FFF])
```

Comment:
I'm not sure this is the best approach to address the problem.
Adding "|[\u0800-\u9FFF]" will cause a big performance hit. Wouldn't removing the check for whitespace be a better choice?

Reply:
The desired behaviour is that URLs are extracted if there is a separating whitespace, but also if there isn't one.